feature-libs/asm/components/customer-list/customer-list.component.ts

```diff
@@ -241,6 +254,29 @@ export class CustomerListComponent implements OnInit, OnDestroy {
     this.closeModal(closeValue);
   }
 
+  onKey(event: any) {
+    if (event.key === 'Enter' || event.pointerType === 'mouse') {
+      console.info(this.searchBox.value);
+      this.fetchCustomers();
+    }
+  }
+
+  isRequired(customerEntry?: any, type?: any): boolean {
```

Comment:
what is the type for `customerEntry` and `type`

Reply:
Done. The type of `customerEntry` is `User`, and the type of `type` is `string`. I have already modified the code and specified the concrete types.